client/src/setupProxy.js

```diff
@@ -11,6 +11,8 @@ function config(app) {
   });
   app.use("/api", proxy);
   app.use("/users", proxy);
+  app.use("/pong", proxy);
+  app.use("/pimg", proxy);
```

Comment:
`pimg` with an M?

Reply:
I believe this fetches the images - necessary on a different route due to lambda issues

Reply:
That's it.